client/src/shared/components/Card/index.tsx

```diff
@@ -0,0 +1,29 @@
+interface CardProps {
+  height: string
+  title: string
+  content: string
```

Comment:
- card should be dynamic as possible. By just passing `string` would not be enough.
- it would be better passing a `children` instead

```suggestion
  children: React.ReactNode
```

```
// usage

<Card>
	<main>
		Hello, This is the content!
	</main>
</Card>
```